fhir/fhir_connector.bal

```diff
@@ -20,44 +20,44 @@ import ballerinax/health.base.auth;
 # This connector allows you to connect and interact with any FHIR server
 @display {label: "FHIR Client Connector"}
 public isolated client class FHIRConnector {
-    private final FHIRConnectorConfig & readonly connectorConfig;
+    private final string baseUrl;
+    private final boolean urlRewrite;
+    private final string? replacementURL;
+    private final MimeType mimeType;
     private final http:Client httpClient;
     private final auth:PKJWTAuthHandler? pkjwtHanlder;
     private final http:Client bulkFileHttpClient;
-    private final auth:PKJWTAuthHandler? bulkFilePkjwtHanlder;
+    private final string? fileServerUrl;
```

Comment:
please add comments to describe what these fields mean.

Reply:
done